scripts/quests/crystalWar/Message_on_the_Winds.lua

```diff
@@ -0,0 +1,218 @@
+                end,
+            },
+        },
+    },
+
+    -- Travel to the past and speak to him again, this time in Batallia Downs (S) (I-7).
+    {
+        check = function(player, status, vars)
+            return vars.Prog == 1
+        end,
+
+        [xi.zone.BATALLIA_DOWNS_S] =
+        {
+            ['Romualdo'] = quest:progressEvent(109),
+
+            onEventFinish =
+            {
+                [109] = function(player, csid, option, npc)
+                    quest:begin(player) -- Quest now appears in log
+                    quest:setVar(player, 'Prog', 2)
+                end,
+            },
+        },
+    },
+
+    -- Speak with Childerich in Grauberg (S) at the west side of the house at (E-12).
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED and vars.Prog == 2
+        end,
+
+        [xi.zone.BATALLIA_DOWNS_S] =
+        {
+            ['Romualdo'] = quest:progressEvent(110), -- Optional dialogue
+        },
+
+        [xi.zone.GRAUBERG_S] =
+        {
+            ['Childerich'] = quest:progressEvent(1),
+
+            onEventFinish =
+            {
+                [1] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 3)
+                end,
+            },
+        },
+    },
+
+    -- Check the ??? at J-9.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED and vars.Prog == 3
+        end,
+
+        [xi.zone.GRAUBERG_S] =
+        {
+            ['Childerich'] = quest:progressEvent(2), -- Optional dialogue
+
+            ['qm3'] = quest:progressEvent(3),
+
+            onEventFinish =
+            {
+                [3] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 4)
+                end,
+            },
+        },
+    },
+
+    -- Speak with Childerich.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED and vars.Prog == 4
+        end,
+
+        [xi.zone.GRAUBERG_S] =
+        {
+            ['Childerich'] = quest:progressEvent(4),
+
+            onEventFinish =
+            {
+                [4] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 5)
+                end,
+            },
+        },
+    },
+
+    -- New default text for Childerich
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED and vars.Prog == 5
+        end,
+
+        [xi.zone.GRAUBERG_S] =
+        {
+            ['Childerich'] = quest:progressEvent(5),
+        },
+
+        [xi.zone.BATALLIA_DOWNS_S] =
+        {
+            ['Romualdo'] = quest:progressEvent(111),
+
+            onEventFinish =
+            {
+                [111] = function(player, csid, option, npc)
+                    quest:complete(player)
+                    player:setCharVar("MessageontheWindsPostCS", 1)
```

Comment:
This could more than likely be handled with a quest var just to keep things contained by using the following pattern:
```
if quest:complete(player) then
    quest:setVar(player, 'Option', 1)
end
```
Just need to make sure its cleaned up like a charvar would (which you're already doing)